Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -150,6 +150,9 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
 
     RefPtr<WebCore::GraphicsLayer> createGraphicsLayer(const String& name, GraphicsLayer::Type);
 
+    WebCore::IntPoint pointInDocumentSpaceForPluginPoint(const WebCore::IntPoint&) const;
```

Comment:
I think these should use standard WebKit naming (like LocalFrameView, and like the existing ones on PDFPluginBase): convertFromXToY(), and _not_ include the type ("point")

Reply:
Made names more standard in [af10ddc](https://github.com/WebKit/WebKit/commit/af10ddcd0e5fdcb907e4c6dc388e54411872b172)